openshift-templates/rolebinding-group/template.yaml

```diff
@@ -0,0 +1,23 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: jenkins-pipeline-no-ocp-triggers
+objects:
+- apiVersion: v1
+  kind: RoleBinding
+  metadata:
+    name: "${ROLE}"
+  roleRef:
+    name: "${ROLE}"                               
+  groupNames:
```

Comment:
@sherl0cks did you find that the `subjects` section wasn't needed when adding the `groupNames` section?

Reply:
it appears that the `subjects` part is generated server side. I'm a bit confused on why it's there to be honest, as it appears to be duplicative info

Reply:
I did find a typo though... fixed and rerunning CI